resources/views/components/menu/menu-sezione.blade.php

```diff
@@ -1,13 +1,26 @@
 @props(['role', 'page'])
 
 @foreach (config('sections') as $elemento => $pagina)
+
+    @php
+        $dashboardRoutes = ['dashboard', 'register', 'timetable'];
```

Comment:
timetable e register devi toglierli, sennò riporta su dashboard, non so se è una cosa voluta.
register non esiste in realtà, corrisponde a plan come rotta.
timetable invece non va bene lì mi sa.

Reply:
Hai ragione ho sistemato l'array con solo quelle che fanno eccezione